File: Searching Algorithms/Linear_Search.cpp
#include <iostream>

using namespace std;

int linear_search(int arr[], int n,int x)
{
	for(int i=0;i<n;i++)
	{
		if(arr[i]==x)
			return i+1;
    }
        return -1;

}

int main()
{
	int n;
	cout<<"Enter the size of the array:\n";
	cin>>n;
	int arr[n],x;
	cout<<"Enter the array element:\n";
	for(int i=0;i<n;i++)
        cin>>arr[i];
	cout<<"Enter the element to be searched:\n";
	cin>>x;
	int result = linear_search(arr,n,x);
	(result == -1)? cout<<"Element is not present in array"
                 : cout<<"Element is present at index:\n"<<result;
	return 0;
}

File: Searching Algorithms/Linear Search/Linear_Search.py
def linear_search(arr,x):
	n = len(arr)
	for i in range(n):
		if arr[i] == x:
			return i+1
	return -1

print("Enter the array elements:")
arr_list = input().split(' ')
arr = 	[int(i) for i in arr_list]
print("Enter the element to be Searched:")
x = int(input())
result = linear_search(arr,x)
if result == -1:
	print("Element not found!!!")
else:
	print("Element fount at position:",result)
